src/Fabulous.XamarinForms/Views/GestureRecognizers/DragGestureRecognizer.fs

```diff
@@ -0,0 +1,48 @@
+namespace Fabulous.XamarinForms
+
+open System.Runtime.CompilerServices
+open Fabulous
+open Xamarin.Forms
+
+type IDragGestureRecognizer =
+    inherit Fabulous.XamarinForms.IGestureRecognizer
+
+module DragGestureRecognizer =
+    let WidgetKey =
+        Widgets.register<DragGestureRecognizer> ()
+
+    let CanDrag =
+        Attributes.defineBindable<bool> DragGestureRecognizer.CanDragProperty
+
+    let DragStarting =
+        Attributes.defineEvent<DragStartingEventArgs>
+            "DragGestureRecognizer_DragStarting"
+            (fun target -> (target :?> DragGestureRecognizer).DragStarting)
+
+    let DropCompleted =
+        Attributes.defineEvent<DropCompletedEventArgs>
+            "DragGestureRecognizer_DropCompleted"
+            (fun target -> (target :?> DragGestureRecognizer).DropCompleted)
+
+
+[<AutoOpen>]
+module DragGestureRecognizerBuilders =
+    type Fabulous.XamarinForms.View with
+        static member inline DragGestureRecognizer<'msg>(onDragStarting: DragStartingEventArgs -> 'msg) =
+            WidgetBuilder<'msg, IDragGestureRecognizer>(
+                DragGestureRecognizer.WidgetKey,
+                DragGestureRecognizer.DragStarting.WithValue(fun args -> onDragStarting args |> box)
+            )
+
+[<Extension>]
+type DragGestureRecognizerModifiers =
+
+    /// <summary>Sets whether users are allowed to drag</summary>
+    /// <param name="value">true to allow users to drag; otherwise, false</param>
+    [<Extension>]
+    static member inline canDrag(this: WidgetBuilder<'msg, #IDragGestureRecognizer>, value: bool) =
+        this.AddScalar(DragGestureRecognizer.CanDrag.WithValue(value))
+
+    [<Extension>]
+    static member inline onDropCompleted(this: WidgetBuilder<'msg, #IDragGestureRecognizer>, onDropCompleted: 'msg) =
+        this.AddScalar(DragGestureRecognizer.DropCompleted.WithValue(fun _ -> onDropCompleted |> box))
```

Comment:
When it's only `'msg`, you can directly do `DropCompleted.WithValue(onDropCompleted)` (see TapGestureRecognizer for example)

Can you fix the couple of events in this PR following the same pattern?

Reply:
I think on the **TapGestureRecognizer** works is because it is define as a Attributes.defineEventNoArgs . But in this case it HAS an EventArgs ?

<img width="1000" alt="Screenshot 2022-02-11 at 08 32 19" src="https://user-images.githubusercontent.com/31915729/153552869-5e64c7eb-b080-4e9b-8648-978249f1587f.png">

Reply:
Oh right. There is an event args but it's useless 😅